layouts/_default/quiz-results.html

```diff
@@ -0,0 +1,127 @@
+{{ define "body-class" }}
+  font-sans
+{{ end }}
+{{ define "html-class" -}}
+  scroll-smooth
+{{- end }}
+
+{{ define "main" }}
+  {{ with .Page.RenderString (.Param "takeover") }}
+    {{ . }}
+  {{ else }}
+    <header class="bg-white px-5 pb-4 pt-8 text-black">
+      <div class="mx-auto max-w-screen-lg">
+        {{ partial "tw/breadcrumbs.html" . }}
+      </div>
+      <div class="mx-auto w-full max-w-2xl">
+        <h1
+          class="font-sans text-3xl font-bold leading-tight text-black sm:text-4xl sm:leading-tight lg:text-5xl lg:leading-tight"
+        >
+          {{- .Title -}}
+        </h1>
+      </div>
+    </header>
+  {{ end }}
+
+
+  <section class="bg-white font-sans">
+    {{ if or (.Content) (.Page.Params.subhed) }}
+      <div
+        class="mx-auto max-w-2xl px-5 pb-10 text-lg font-medium md:px-0 lg:px-0"
+      >
+        {{ with .Page.Params.subhed }}
+          <h2 class="mb-5 text-5xl font-black uppercase">{{ . }}</h2>
+        {{ end }}
+        {{ with .Content }}
+          <div class=" article-content">
+            {{ . }}
+          </div>
+        {{ end }}
+      </div>
+    {{ end }}
+    {{ with .Page.RenderString (.Param "newsletter") }}
+      <div class=" font-sans">
+        {{ . }}
+      </div>
+    {{ end }}
+
+    {{ .Page.RenderString (.Param "testimonials-group") }}
+
+    {{ with .Page.RenderString (.Param "supplemental") }}
+
+      <article
+        class=" article-content mx-auto max-w-screen-xl px-5 py-24 font-sans md:grid md:grid-cols-12 md:grid-rows-[auto_1fr] md:gap-x-5 md:gap-y-3"
+      >
+        {{ . }}
+      </article>
+    {{ end }}
+    {{ range $i, $blocks := .Params.blocks }}
+      {{ $id := urlize .title }}
+      <div
+        id="{{ $id }}"
+        x-data
+        x-intersect.half="$dispatch('intersect:support', $el.id)"
+        data-ga-category="blocks"
+        class=" w-full"
+      >
+        {{ partial "blocks/block" (dict
+          "Page" $
+          "block" .
+          )
+        }}
+      </div>
+    {{ end }}
+    {{ with .Page.RenderString (.Param "answer-key") }}
+      <div
+        x-data="{ active: 1 }"
+        class="mx-auto max-w-2xl space-y-4 px-5 py-12 lg:px-0"
+      >
+        <h2 class="leading-tight; mb-4 text-3xl font-bold">Answer Key</h2>
```

Comment:
Scoot this a little closer to the "Meet the Candidates" button. I almost didn't see the key at first because it's further down the page and I expected there to just be story promos down there.

Reply:
Another solution is maybe to make "Answer Key" closer and colorful or decorated so it grabs the eye and you notice the key.